src/systems/systems_wasm/src/electrical.rs

```diff
@@ -35,60 +33,49 @@ pub(super) fn electrical_buses<const N: usize>(
 /// The default MSFS APU is still used during engine start.
 /// At this moment, the engines cannot be started without it.
 /// Once pneumatics and the engine model are completed, this
-/// type can probably be removed.
-pub(super) struct MsfsAuxiliaryPowerUnit {
-    is_available_id: VariableIdentifier,
-    msfs_apu_is_on: AircraftVariable,
+/// function can probably be removed.
+pub(super) fn auxiliary_power_unit(
+    is_available_variable: Variable,
     fuel_valve_number: u8,
```

Comment:
@BlueberryKing I guess this relates to what we talked earlier that bleed still not plugged into engines right?

Reply:
Yeah, exactly